mimic/rest/nova_api.py

```diff
@@ -308,3 +315,150 @@ def get_ips(self, request, tenant_id, server_id):
                 request, server_id
             )
         )
+
+    @app.route('/v2/<string:tenant_id>/servers/<string:server_id>/metadata',
+               branch=True)
```

Comment:
I think if branch=`True`, everything gets passed to the next resource?  See https://github.com/rackerlabs/mimic/blob/master/mimic/resource.py#L43